library/init/meta/interactive.lean

```diff
@@ -6,8 +6,11 @@ Authors: Leonardo de Moura
 prelude
 import init.meta.tactic init.meta.rewrite_tactic init.meta.simp_tactic
 import init.meta.smt.congruence_closure init.category.combinators
+import init.meta.lean_parser
 
 namespace interactive
+@[reducible] meta def parse {α : Type} (p : lean_parser α) : Type := α
```

Comment:
I would hate to give up the strong typing. Can we convert the parsed `vm_obj`s to `ts_vm_obj`s and then wrap them in a macro? There should never a need to unfold it, but I believe we would need a new VM opcode for loading from a `ts_vm_obj`.